tests/models/encoder_decoder/language/test_bart.py

```diff
@@ -2,170 +2,12 @@
 
 Run `pytest tests/models/encoder_decoder/language/test_bart.py`.
 """
-from typing import List, Optional, Tuple, Type
-
 import pytest
```

Comment:
we can drop this folder altogether, `../language/language` was never really great